### PR TITLE
Adding fuel content

### PR DESCRIPTION
**Pull request by nkoenig**
Signed-off-by: Nate Koenig <nate@openrobotics.org>

# 🎉 New feature

Moving the content from [Freshdesk](https://openrobotics.freshdesk.com/support/home) to here.

Requires https://github.com/gazebo-web/gazebosim-web-backend/pull/6

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: fuel/contributing_model.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Contributing a New Model
+
+The easiest way to contribute a new model is via [https://app.gazebosim.org/fuel/models/upload](https://app.gazebosim.org/fuel/models/upload).
+
+Fuel assumes that a model conforms to Gazebo's model structure. At minimum,
```
Comment (mabelzhang) [inline fuel/contributing_model.md:5]: Is it useful/maintainable to link to this more detailed tutorial? https://gazebosim.org/api/sim/7/meshtofuel.html

File: fuel/contributing_world.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Contributing a New World
+
+A world consists of a description of global properties and a set of models
+that populate the environment. The description format is SDF, and you can
+find example worlds at [https://app.gazebosim.org/fuel/worlds](https://app.gazebosim.org/fuel/worlds). Worlds can be used by simulators to launch and simulate models in a specific environment.
+
+You can upload a new world to Gazebo by complete the form on [https://app.gazebosim.org/fuel/worlds/upload](https://app.gazebosim.org/fuel/worlds/upload). The upload form allows you to specify a name for the world, along with a description and tag data.
```
Comment (mabelzhang) [inline fuel/contributing_world.md:7]:
```suggestion
You can upload a new world to Gazebo by completing the form on [https://app.gazebosim.org/fuel/worlds/upload](https://app.gazebosim.org/fuel/worlds/upload). The upload form allows you to specify a name for the world, along with a description and tag data.
```

File: fuel/contributing_world.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Contributing a New World
+
+A world consists of a description of global properties and a set of models
+that populate the environment. The description format is SDF, and you can
```
Comment (mabelzhang) [inline fuel/contributing_world.md:4]:
```suggestion
that populate the environment. The description format is [SDF](http://sdformat.org/), and you can
```

File: fuel/copyright.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+2. A description of your work that you believe has been infringed
+
+In your complaint, be sure to clearly and completely describe the copyrighted content you are seeking to protect. If multiple copyrighted works are covered in your complaint, the law allows a representative list of such works.
+
+3. Each allegedly infringing URL
+
+Your complaint must contain the specific URL of the content you believe infringes your rights, or we will be unable to locate it. General information about the location of the content is not adequate. Please include the URL(s) of the exact content at issue.
+
+4. You must agree too and affirm both of the following statements:
```
Comment (mabelzhang) [inline fuel/copyright.md:64]:
```suggestion
4. You must agree to and affirm both of the following statements:
```

File: fuel/data_privacy_policy.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+service, and we do not share or distribute private information.
+
+
+## Information you give to us. 
+
+This includes assets, such as 3D meshes, as well as some user information,
+such as email addresses during the sign up process. We explicitly do
+not store passwords. The user information we collect uniquely
+identifies a person, and is required by many of the features we
+provided by this web service.
```
Comment (mabelzhang) [inline fuel/data_privacy_policy.md:21]:
```suggestion
provide by this web service.
```

File: fuel/data_privacy_policy.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+information is used.
+
+## Information we collect
+
+We are not striving to be an information broker. To that end, we securely
+store the minimal amount of information necessary to create a functional
+service, and we do not share or distribute private information.
+
+
+## Information you give to us.
```
Comment (mabelzhang) [inline fuel/data_privacy_policy.md:15]: This is technically not a complete sentence and shouldn't be followed by a period
```suggestion
## Information you give to us
```

File: fuel/data_privacy_policy.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+## Information you give to us. 
+
+This includes assets, such as 3D meshes, as well as some user information,
+such as email addresses during the sign up process. We explicitly do
+not store passwords. The user information we collect uniquely
+identifies a person, and is required by many of the features we
+provided by this web service.
+
+
+## Information we get from your use of our services.
```
Comment (mabelzhang) [inline fuel/data_privacy_policy.md:24]:
```suggestion
## Information we get from your use of our services
```

File: fuel/data_privacy_policy.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+This information is used to provide a better user-experience. We do not sell
+or use this information for ads or marketing.
+
+
+## Inappropriate Content
+
+Content may be immediately removed if it
+
+  * contains images that do not represent the actual data,
+  * violates copyright law ( See our [Copyright](/docs/latest/fuel_copyright) and [Fair Use](/docs/latest/fuel_fair_use) pages), or
```
Comment (mabelzhang) [inline fuel/data_privacy_policy.md:36]:
```suggestion
  * violates copyright law (See our [Copyright](/docs/latest/fuel_copyright) and [Fair Use](/docs/latest/fuel_fair_use) pages), or
```

**Review by mabelzhang — COMMENTED** [review]
This was educational for me.
Minor comments.
Haven't run the backend PR yet. I think last (and only) time I did something in the backend repo, I wasn't sure how to get the docs to show up in a browser locally...

**Review by mabelzhang — COMMENTED** [review]
Returning to this now that I can preview locally. A few comments on formatting.

File: fuel/contributing_model.md (hunk 1 above)
Comment (nkoenig) [reply fuel/contributing_model.md]: Thanks. Added in f1c6049b645e575f39a66c4eee2fb47884129079

File: fuel/content_deletion_policy.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+deletion, or deletion of inappropriate content.
+
+We place few restrictions when adding content to this service, however
+deletion of content can cause downstream harm. For example, another user or
+organization may depend on a model you have added. Deletion of your model
+can then lead to breaking changes. We adhere to the following rules in order
+to strike a balance between content deletion and the needs of other users.
+
+  1. You can delete any content that you have created and uploaded, but the content will remain in a soft-deleted state for one month.
+    * **Soft-delete** Is when content is marked as deleted, but remains available for download.
```
[the remark this answers is not in the document]
Comment (nkoenig) [reply fuel/content_deletion_policy.md]: I had to add a few spaces. See f1c6049b645e575f39a66c4eee2fb47884129079

File: fuel/about.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+
+Fuel is a two-part web application for hosting and accessing simulation
+assets. The first part is this website, which displays assets in a human
+consumable manner. The second part is a backend server that manages the
+simulation assets. See the Using models with Gazebo, and REST API sections
+below for information.
+
+## Support
+
+For technical questions and help on using Fuel, please use the [https://answers.gazebosim.org](forum).
```
[the remark this answers is not in the document]
Comment (nkoenig) [reply fuel/about.md]: I've also changed this to point to robotics.stackexchange.com

File: fuel/about.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# About Fuel
+
+Fuel is a two-part web application for hosting and accessing simulation
+assets. The first part is this website, which displays assets in a human
```
[the remark this answers is not in the document]
Comment (nkoenig) [reply fuel/about.md]: Fixed to point to app.gazebosim.org

File: fuel/about.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+consumable manner. The second part is a backend server that manages the
+simulation assets. See the Using models with Gazebo, and REST API sections
```
[the remark this answers is not in the document]
Comment (nkoenig) [reply fuel/about.md]: I've updated this to link to the REST API, and removed the "Using models with Gazebo" wording.

**Review by mabelzhang — APPROVED** [review]

**Pull request merged**